### PR TITLE
docs(readme): Add link to run-komac action

### PR DESCRIPTION
**Pull request by michidk**
Adds a link to my [run-komac](https://github.com/michidk/run-komac/) action.

This action installs the specified version into the GHA job.

### CODE REVIEW

**Comment by russellbanks** [comment]
@michidk, are you happy with the change I've made?

File: README.md [hunk 1]
```diff
@@ -136,6 +136,10 @@ jobs:
           # installers-regex: '\.exe$' # Only .exe files
 ```
 
+### Alternative actions 🔄
+
+- Run Komac manually: [michidk/run-komac](https://github.com/michidk/run-komac)
```
Comment (michidk) [inline README.md:141]:
```suggestion
- Release automation, which can be used with external repositories: [michidk/winget-updater](https://github.com/michidk/winget-updater)
- Run Komac manually: [michidk/run-komac](https://github.com/michidk/run-komac)
```

Comment (michidk) [reply README.md:141]: @russellbanks Nice approach with the list. Also added my release automation script. What do you think?

**Comment by michidk** [comment]
Looks good, thanks!

**Review by russellbanks — APPROVED** [review]

**Pull request merged**